packages/gatsby/src/schema/node-model.js

```diff
@@ -24,6 +24,7 @@ import { GatsbyIterable, isIterable } from "../datastore/common/iterable"
 import { reportOnce } from "../utils/report-once"
 import { wrapNode, wrapNodes } from "../utils/detect-node-mutations"
 import { toNodeTypeNames, fieldNeedToResolve } from "./utils"
+import withResolverContext from "./context"
```

Comment:
```suggestion
import { withResolverContext } from "./context"
```

Reply:
it has a default export https://github.com/gatsbyjs/gatsby/blob/9d33b107d167e3e9e2aa282924a0c409f6afd5a0/packages/gatsby/src/schema/context.ts#L14